README.md

```diff
@@ -0,0 +1,15 @@
+# API CRUD
+
+A small demonstration of an implementation of a typical API CRUD interface, written in TypeScript. It uses Fastify and Knex for routing and DB management, Swagger and Jest for documentation and tests.
+
+## How to use it
+
```

Comment:
### Prerequisite
Have `node`, `docker`, `docker-compose` and `npm` or `yarn` installed.

### First steps
#### Install dependencies
First run, you need to install dependencies.

Run (for `npm`):
- `npm install` (or `npm i`)

Run (for `yarn`):
- `yarn install`

#### Build docker image
Then need to build docker image:
- `sudo docker-compose build`

#### Start docker images
Start docker images:
`sudo docker-compose up` (with the flag `-d` if you want to run it in *Daemon* mode)

#### To use
The server will be listening at `https://127.0.0.1:8080` and you will have the Swagger documentation page at the root.